src/BinaryParsers/ELFBinary/Dwarf/LEB128Helper.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.BinaryParsers.Dwarf
+{
+    internal static class LEB128Helper

```

Comment:
>LEB [](http://example.com/codeflow?start=26&length=3)

question, as the class name is LEB, do we need to check if the file is ELFSharp.Endianess.LittleEndian at some point

Reply:
all utility code are being replaced.